### ISSUE TITLE
About Response data in DELETE

### ISSUE THREAD

**Issue by zz-zye** [swsnu/swppfall2021#135]
Hello, sorry for the late question.
In DELETE method of `/api/article/id/` and `/api/comment/id/`, do we need to return the data of the deleted article or comment inside HttpResponse?

Thank you.

**Comment by ttoru96** [comment]
No, you don't need to return the data of the deleted article or comment inside HttpResponse. 

You don't need to care about what is not covered in spec. If you refer to spec, response for DELETE method is not specified unlike other methods.
